docs/howto/operator/gcp/natural_language.rst

```diff
@@ -0,0 +1,179 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Google Cloud Natural Language Operators
+=======================================
+
+The `Google Cloud Natural Language <https://cloud.google.com/natural-language/>`__
+can be used to reveal the structure and meaning of text via powerful machine
+learning models. You can use it to extract information about people, places,
+events and much more, mentioned in text documents, news articles or blog posts.
+You can use it to understand sentiment about your product on social media or
+parse intent from customer conversations happening in a call center or a
+messaging app.
+
+.. contents::
+  :depth: 1
+  :local:
+
+Prerequisite Tasks
+^^^^^^^^^^^^^^^^^^
+
+.. include:: _partials/prerequisite_tasks.rst
+
+
+.. _howto/operator:CloudNaturalLanguageDocuments:
+
+Documents
+^^^^^^^^^
+
+Each operator use a :class:`~google.cloud.language_v1.types.Document` for
+representing text.
+
+Here is an example of document with text provided as a string:
+
+.. literalinclude:: ../../../../airflow/contrib/example_dags/example_gcp_natural_language.py
```

Comment:
This may just be a problem with rendering in the PR, but the literal includes are not working for me:
https://github.com/apache/airflow/blob/b66ea5b2a1dd87572e2ac5626fce9ea164bd8097/docs/howto/operator/gcp/natural_language.rst#id7

Reply:
Github don't support all sphinx directive. Full preview is available: https://github.com/apache/airflow/pull/4980#issuecomment-478105100